rdkit/Chem/Fraggle/FraggleSim.py

```diff
@@ -277,7 +272,7 @@ def partialSimilarity(atomID):
   # generate mol object & fp for input mol (we are interested in the bits each atom sets)
   pMol = Chem.Mol(mol)
   aBits = []
-  _ = Chem.RDKFingerprint(pMol, atomBits=aBits, **rdkitFpParams)
+  Chem.RDKFingerprint(pMol, atomBits=aBits, **rdkitFpParams)
```

Comment:
See Greg's comment in previous PR. He prefers to keep the assignment to underscore to indicate that ignoring the result is on purpose

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.